.github/workflows/lint.yml

```diff
@@ -18,20 +18,18 @@ jobs:
     name: lint
     runs-on: ${{ matrix.os }}
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
 
       - name: Set up Go
-        uses: actions/setup-go@v3
+        uses: actions/setup-go@v5
         with:
           go-version-file: './go.mod'
           check-latest: true
 
       - run: make deps
 
       - name: golangci-lint
-        uses: golangci/golangci-lint-action@v3
-        with:
-          skip-pkg-cache: true
```

Comment:
Option is no longer needed, and was removed in v5: https://github.com/golangci/golangci-lint-action?tab=readme-ov-file#compatibility